
w/ XFree86 Radeon hardware acceleration (at least on iBook) the
NV extension does produce major pixel garbage.

  - Rene Rebe <rene@rocklinux.org>

--- evas-2004-02-25/src/lib/engines/gl_common/evas_gl_context.c.orig	2004-02-27 03:12:11.000000000 +0100
+++ evas-2004-02-25/src/lib/engines/gl_common/evas_gl_context.c	2004-02-27 03:12:49.000000000 +0100
@@ -64,7 +64,7 @@
 	if (ext)
 	  {
 	     if (strstr(ext, "GL_SGIS_generate_mipmap")) gc->ext.sgis_generate_mipmap = 1;
-	     if (strstr(ext, "GL_NV_texture_rectangle")) gc->ext.nv_texture_rectangle = 1;
+	     if (strstr(ext, "GL_NV_texture_rectangle")) gc->ext.nv_texture_rectangle = 0;
 	     /* technically this should work, as its a compatible */
 	     /* implementation of the nvidia texture_rectangle extension */
 	     /* since the #define value is the same as is the description */
